src/test/regress/output/multi_insert_select_behavioral_analytics_create_table.source

```diff
@@ -62,3 +62,370 @@ CREATE INDEX is_index3 ON users_table(value_1);
 CREATE INDEX is_index4 ON events_table(event_type);
 CREATE INDEX is_index5 ON users_table(value_2);
 CREATE INDEX is_index6 ON events_table(value_2);
+-- create a helper function to create types/functions on each node
+CREATE FUNCTION run_command_on_master_and_workers(p_sql text)
```

Comment:
Isn't this same with `run_command_on_coordinator_and_workers ()`? Maybe we can move it to a more central place such citus tools or another base regression tests and use it?

https://github.com/citusdata/citus/blob/40ebd93be3b0140155dc4d24836033d0d917b542/src/test/regress/sql/multi_schema_support.sql#L742